includes/Context.php

```diff
@@ -140,6 +140,33 @@ public function get_reference_site_url() {
 		return $site_url;
 	}
 
+	/**
+	 * Gets the permalink of the reference site to use for stats.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @param int|\WP_Post $post  Optional. Post ID or post object. Default is the global `$post`.
+	 *
+	 * @return string Reference permalink.
+	 */
+	public function get_reference_permalink( $post = 0 ) {
+		$orig_site_url      = untrailingslashit( home_url() );
+		$reference_site_url = untrailingslashit( $this->get_reference_site_url() );
+
+		// Specific on homepage get_permalink either return the global posts or false.
+		if ( is_home() ) {
```

Comment:
This condition should not be here. The `$post` parameter is passed so that should be used.

Furthermore `is_home()` is only the actual home page for blogs, not for sites that have a dedicated front page.